client/src/helpers/APICalls/updateProfile.ts

```diff
@@ -0,0 +1,16 @@
+import { ProfileApiData } from '../../interface/ProfileApiData';
+import { FetchOptions } from '../../interface/FetchOptions';
+
+const updateProfile = async (values: ProfileApiData): Promise<ProfileApiData> => {
+  console.log('The values are: ', JSON.stringify({ values }));
```

Comment:
This can also be removed, make sure to clean up PRs of unnecessary console logs going forward.

Reply:
OK